src/services/walkthroughServices.js

```diff
@@ -1,9 +1,9 @@
-import { list, create, watch, currentUser, OpenShiftWatchEvents } from './openshiftServices';
+import { list, create, remove, watch, update, currentUser, OpenShiftWatchEvents } from './openshiftServices';
 import { walkthroughTypes } from '../redux/constants';
 import { FULFILLED_ACTION } from '../redux/helpers';
-import { buildServiceInstanceResourceObj } from './serviceInstanceServices';
+import { buildServiceInstanceResourceObj, DEFAULT_SERVICES } from './serviceInstanceServices';
 
-const WALKTHROUGH_SERVICES = ['fuse', 'che', 'launcher', 'enmasse-standard'];
+const WALKTHROUGH_SERVICES = ['fuse', 'che', 'launcher', 'enmasse-standard', 'amq-broker-71-persistence'];
```

Comment:
should we reference these from the consts in serviceInstanceServices.js rather than having the names in 2 places?

e.g. `DEFAULT_SERVICES.ENMASSE`, `DEFAULT_SERVICES.AMQ` etc...

Reply:
Good catch, yep